database/migrations/2020_05_07_100000_create_refunds_table.php.stub

```diff
@@ -0,0 +1,26 @@
+2020_05_07_100000_create_refunds_table.php<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRefundsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('refunds', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('payment_id')->references('id')->on('payments');
+            $table->string('refund_number')->index()->unique(); // Generated by system. This is identifier used to communicate with customers about their refund.
+            $table->unsignedInteger('amount'); // Amount is in cents.
+            $table->string('method');
+            $table->string('transaction_number')->nullable();
+            $table->foreignId('voucher_id')->nullable()->references('id')->on('vouchers');
+            $table->foreignId('issuer_id')->nullable()->references('id')->on('users');
+            $table->foreignId('creator_id')->references('id')->on('users');
+            $table->foreignId('updater_id')->references('id')->on('users');
+            $table->dateTime('issued_at')->nullable();
+            $table->timestamps();
+        });
+    }
+}
```

Comment:
👍

Reply:
Should all of the table names referenced in the `foreignId` definitions be pulled from model classes in config?  Or, are you ok with packages using explicit table names for tables owned by other packages?

```
            $userModel = config('discounts.model.user');
            $userTable = (new $userModel)->getTable();

            $table->foreignId('creator_id')->references('id')->on($userTable);
            $table->foreignId('updater_id')->references('id')->on($userTable);
```

Reply:
I added it to rules file.

Reply:
Yes, thank you both.